pkg/conda/meta/testdata/README.md

```diff
@@ -0,0 +1,30 @@
+To recreate test files, run:
+```bash
```

Comment:
I think we can use [miniconda3](https://hub.docker.com/r/continuumio/miniconda3) image to recreate test files:
```suggestion
```bash
docker run --name miniconda --rm -it continuumio/miniconda3@sha256:58b1c7df8d69655ffec017ede784a075e3c2e9feff0fc50ef65300fc75aa45ae bash
```

Reply:
Recreated the test files based on the miniconda image